test/controllers/v1/relay_controller_test.exs

```diff
@@ -95,6 +98,72 @@ defmodule Cog.V1.RelayControllerTest do
     assert updated["description"] == @update_attrs.description
   end
 
+  test "relays are enabled in more than just name", %{authed: requestor} do
+    # We create a relay and add a bundle to it so we can query for it in
+    # 'Cog.Relay.Relays'
+    relay = relay("relay-enable-relay-1", "foobar")
+    bundle = bundle("relay-enable-bundle-1")
+    relay_group = relay_group("relay-enable-group-1")
+    add_relay_to_group(relay_group.id, relay.id)
+    assign_bundle_to_group(relay_group.id, bundle.id)
```

Comment:
How about making some helper methods that can do all this in one shot, like we now do with `cogctl`.

I definitely think we should still have the fine-grained helpers that you've got here, but I think it'd help the tests to put a lot of that tedious setup logic behind some well-named helpers that can do it all in one shot.